packages/api/cms-api/src/ml/ml.types.ts

```diff
@@ -0,0 +1,30 @@
+export interface Options {
+    seed?: number;
+    temperature?: number;
+    maxTokens?: number;
+}
+
+export type ModelType = "text" | "textAdvanced" | "image" | "imageAdvanced";
+
+export interface MlRequest {
+    prompt?: string;
+    instructions?: string;
+    context?: string;
+    examples?: Example[];
+    options?: Options;
+    image?: Image;
+}
+
+export interface MlResponse {
+    response: string;
+    image?: string;
+}
+
+export interface Example {
+    role: "user" | "assistant";
+    text: string;
+}
+
+export interface Image {
+    base64: string;
+}
```

Comment:
Should we prefix this with MlRequest?

Reply:
Like MLRequestImage?

Reply:
Yes, either this, or don't export the types that are not need. I believe only ContentGenerationRequest is needed as export.